### ISSUE TITLE
add `blade_%s:` prefix to geometry variables

### ISSUE THREAD

**Issue by fzahle** [FUSED-Wind/fusedwind-dev#21]
The blade planform is defined with the following parameters: `['x', 'y', 'z', 'rot_x', 'rot_y', 'rot_z', 'chord', 'rthick', 'p_le']`. In the old OpenMDAO they were nested inside a vartree, making it clear that all these parameters belonged to a planform definition. Now that vartrees no longer exist, the first version we did simply had the parameters added and promoted with the above names. While names like `chord` are self-explanatory `x y z` are not. so to be more unambiguous I propose that we start using prefixes for these parameters, so they're referred to as e.g. `blade_ae:s` and `blade_ae:chord`. The pf parameters discretized for the structural calculations will then be named `blade_st:s` and `blade_st:chord`.

Any objections? @mrosemeier what do you think?

**Comment by mrosemeier** [comment]
I have no objections using prefixes. Would that more or less replace the suffixes that are currently added in the components, i.e. `suffix='_st'`?

**Comment by fzahle** [comment]
yes, the suffix in the current version will be become part of the prefix.

**Comment by fzahle** [comment]
FIY @tlbl and @pilotmm let me know if you have any comments.

And I won't merge this until we have adapted the other wrappers.

**Comment by pilotmm** [comment]
I think that change makes a lot of sense...
